### ISSUE TITLE
Discord Support Server

### ISSUE THREAD

**Issue by Isaiahmag** [smartfrigde/armcord#226]
**Describe the bug**
Invite links you to a fake support server and am unsure if there is even a real one.

**To Reproduce**
1. Go to 'Tray icon'
2. Click on 'Discord support server'
3. See 'Fake discord invite screen'

**Expected behavior**
A real invite!

**Desktop (please complete the following information):**
 - OS: Linux
 - Method of installation: AUR
 - Version: 3.0.8

**Comment by smartfrigde** [comment]
What do you mean by fake invite screen?
https://discord.gg/6rSPDdYz here's the invite if you don't wanna join through that button but all it does is load up invite url

**Comment by Isaiahmag** [comment]
> What do you mean by fake invite screen? https://discord.gg/6rSPDdYz here's the invite if you don't wanna join through that button but all it does is load up invite url

Oh sorry about that but for some reason it was actually opening another window and when I used inspect element it said some stuff related to a scam but currently is not doing that while in the server. No joke though without having a server there was 2 windows that would pop up and technically I wouldn't be able to access my Discord account unless I rebooted ArmCord after clicking the button in the tray. I guess I needed to have actual Discord to open to access the server as normally through that button I couldn't.
